requirements.txt

```diff
@@ -4,4 +4,6 @@ git+https://github.com/flesser/django-simple-history@django2.0
 psycopg2==2.7.3
 pytz==2017.3
 djangorestframework==3.7.7
+fuzzywuzzy

```

Comment:
Please add the versions you are using, otherwise the new versions may break the system

Reply:
ok